pkg/apis/toolchain/v1alpha1/tiertemplate_types.go

```diff
@@ -0,0 +1,46 @@
+package v1alpha1
+
+import (
+	templatev1 "github.com/openshift/api/template/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// TierTemplateSpec defines the desired state of TierTemplate
+// +k8s:openapi-gen=true
+type TierTemplateSpec struct {
+	
+	// The type of the template. For example: "code", "dev" or "stage"
```

Comment:
I know that it lists only the examples, but I would add also the "cluster" type
```suggestion
	// The type of the template. For example: "code", "dev", "stage" or "cluster"
```